doc/examples/transform/plot_masked_register_translation.py

```diff
@@ -33,23 +33,23 @@
 image = data.camera()
 shift = (-22, 13)
 
-corrupted_pixels = np.random.choice([False, True], 
-                                    size = image.shape, 
+corrupted_pixels = np.random.choice([False, True],
+                                    size = image.shape,
```

Comment:
This looks bad 😄. Also, the same issue below. I'll fix that once I review that PR history.